packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -499,7 +504,7 @@ export class AppStudioPluginImpl {
       );
     }
 
-    const outlineFile = `${appDirectory}/${manifest.icons.outline}`;
+    const outlineFile = `${resourcesDir}/${manifest.icons.outline}`;
```

Comment:
the outline file path is already the relative path to `resourceDir`, so you cannot prepend `resourceDir` here.
![Screenshot from 2021-09-07 16-27-35](https://user-images.githubusercontent.com/9698542/132311676-7259cf1c-eb87-4b3e-ae6b-6f2227d979a9.png)

Reply:
Nice catch! it affect the azure project indeed. Will fix it soon.